tools/run-wpt.mjs

```diff
@@ -62,17 +64,25 @@ if (
   usage();
   process.exit(0);
 }
+const restArgs = process.argv.slice(2);
+
+// Weather to run the tests. `false` value is useful for updating expectations
+// based on the WPT report.
+const RUN_TESTS = process.env.RUN_TESTS || 'true';
```

Comment:
let's convert env variables into predictable booleans instead of strings, e.g. `(process.env.RUN_TESTS ?? 'true') === 'true'`

Reply:
This requires updating all the variables to prevent confusion, so I'd keep it out of scope of the PR